media/stump-radio/stump-radio.asd

```diff
@@ -0,0 +1,6 @@
+(asdf:defsystem #:stump-radio
+  :description "mplayer-based radio for stumpvm"
```

Comment:
Did you mean "stumpwm"?

Reply:
haha, not the first time I mistyped that one... Thanks